src/qt/createwalletdialog.cpp

```diff
@@ -40,6 +40,10 @@ CreateWalletDialog::CreateWalletDialog(QWidget* parent) :
     ui->descriptor_checkbox->setEnabled(false);
     ui->descriptor_checkbox->setChecked(false);
 #endif
+#ifndef USE_BDB
+    ui->descriptor_checkbox->setEnabled(false);
+    ui->descriptor_checkbox->setChecked(true);
+#endif
```

Comment:
Since descriptor wallets are still experimental, I think it might be better to leave the checkbox alone (enabled + default off) but make the button give an error until the user explicitly checks it?

Reply:
I think that UX would be frustrating to use because there's only one true option in that case, but you're pretending there's 2. Users will still get the experimental warning when the wallet is created.